src/main/webapp/scripts/civic.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Extract the government officials from the civic API response.
+ */
+function extractOfficials(civicObj) {
+  let officials = new Object();
+  let levels = ["administrativeArea1", "administrativeArea2", "country", "international", "locality", "regional", "special", "subLocality1", "subLocality2"];
+  levels.forEach((level) => {
+    officials[level] = new Array();
+  });
+  civicObj.offices.forEach((officeObj) => {
```

Comment:
Could use some commenting in the block below this, it was a bit hard for me to figure out what was going on.